src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -249,6 +249,7 @@ class BasicSettings extends React.Component {
         // when changing Provision type to ISO, set the default time zone according to the OS
         if (changes.provisionSource === 'iso') {
           changes.timeZone = this.checkTimeZone(changes.operatingSystemId, changes.templateId)
```

Comment:
1. the difference between `timeZone` and `initTimezone` is not clear to me. Perhaps some comment would help?
2. `initTimezone` sounds more like a method to me. For this reason I would prefer the longer form `cloudInitTimeZone`.
3. this is a minor one but we could standardize the spelling either to `timeZone` or just to `timezone` (both forms are correct English so it's just consistency)

Reply:
Well, the more I think about it, the less sure I am, too :D Maybe it worth to look for example here: https://github.com/oVirt/ovirt-web-ui/blob/master/src/ovirtapi/transform.js#L228
Or here:
![aaa](https://user-images.githubusercontent.com/13417815/92722693-3f28ac80-f368-11ea-9fbf-97e8fd6550ad.png)
![aa](https://user-images.githubusercontent.com/13417815/92722701-43ed6080-f368-11ea-8fa8-ef4b9fa5075c.png)
![a](https://user-images.githubusercontent.com/13417815/92719099-f4586600-f362-11ea-8362-b2066d27ea8e.png)

Note that there isn't any possibility to set timezone when cloud init enabled, in the Basic settings, only sysprep.

Reply:
We can discuss the use of `cloudInitTimeZone` vs. `initTimezone`, timeZone` vs. `timezone` forms but this is not the part of this PR. We still can rename it later. And the result of the topic mentioned [here](https://github.com/oVirt/ovirt-web-ui/pull/1288#discussion_r486250035) is also related to this.